Gotify.json

```diff
@@ -0,0 +1,39 @@
+{
+    "Gotify": {
+        "description": "A simple server for sending and receiving messages. <p>Based on the official Gotify x86 docker image: <a href='https://hub.docker.com/r/gotify/server' target='_blank'>https://hub.docker.com/r/gotify/server</a>.</p>",
+        "ui": {
+            "slug": ""
+        },
+        "website": "https://gotify.net/",
+        "version": "latest",
+        "containers": {
+            "gotify": {
+                "image": "gotify/server",
+                "launch_order": 1,
+                "ports": {
+                    "80": {
+                        "description": "Gotify server port. Suggested default: 80",
+                        "host_default": 80,
+                        "label": "Server port",
+                        "protocol": "tcp",
+                        "ui": true
+                    }
+                },
+                "volumes": {
+                    "/app/data": {
+                        "description": "Choose a Share for the database, plugins and images for applications.  Eg: create a Share called gotify for this purpose.",
+                        "label": "Database and App Images",
+                        "min_size": 1073741824
+                    }
+                },
+                "environment": {
```

Comment:
No need for setting the TZ variable here, I believe, as Rockstor automatically binds the `/etc/localtime` volume in all containers:
https://github.com/rockstor/rockstor-core/blob/3ad7eee27a40badab7edeea6dc198e800f6b121e/src/rockstor/storageadmin/views/rockon_helpers.py#L187-L188

Reply:
I'm not familiar enough with Alpine to know if that actually works there. From quick googling it seems that's not always enough for Alpine?

Reply:
Oh good point... I'll have to try that, then. Thanks for pointing that out.

Reply:
Thank you for looking into it. Somehow I had come to the conclusion that the image was alpine based.